l10n_fr_ecotaxe/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'France Custom Ecotaxe',
+    'summary': 'Use Ecotaxe in French localisation contexte',
+    'version': '8.0.0.1.0',
+    'author': "Akretion,Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
+    'category': 'Localization/Account Taxes',
+    'depends': [
+        'account',
+        'product',
+        'sale',
+        'purchase'
+    ],
```

Comment:
if you have account_accountant, you can't modify ecotaxe settings.
If sale, no needs of product.